### DISCUSSION TITLE
Change project or namespace on sub projects

### DISCUSSION THREAD

**Discussion by magicweasel** [JasonRowe/KestrelMock#51] [Ideas]
![image](https://github.com/JasonRowe/KestrelMock/assets/7963468/7d509e1b-9327-471b-8ada-a038ce5e5914)

One thing I found quite awkward was that the sub projects have different namespaces, so new classes went in with the wrong namespace and required correction.

I'd suggest we just change all the namespaces in `KestrelMock` project to `KestrelMock.[Whatever]`?

Alternatively, if there's a reason to keep them as `KestrelMockServer` can we change the namespace in the project so that it's easier to add new files?

(I'm happy to make either change, just checking here for direction first ...)

**Comment by JasonRowe** [comment]
Yeah as long as we don't end up with the following issue again or change the API.

https://github.com/JasonRowe/KestrelMock/issues/21
